src/tests/functions/Add-DscResourceModule.Tests.ps1

```diff
@@ -0,0 +1,63 @@
+Describe 'Vendoring a DSC module' {
+  InModuleScope puppet.dsc {
+    Context 'When only -Name is specified' {
+      Mock New-Item -Verifiable
+      Mock Save-Module -Verifiable
+      Mock Test-Path { $false } -ParameterFilter { $path -eq './tmp' }
+      Mock Get-CanonicalPath { './tmp' }
+      Mock Test-Path { $false } -ParameterFilter { $path -eq './tmp_tmp' }
+      Mock Get-ChildItem
+      Mock Move-Item
+      Mock Remove-Item
+
+      Add-DscResourceModule -Name SomeDSCModule
+
+      It 'downloads the latest version of the module to ./tmp' {
+        Assert-VerifiableMock
+        Assert-MockCalled New-Item -Times 1 -ParameterFilter { ($path -eq './tmp') -and ($ItemType -eq 'Directory') }
+        Assert-MockCalled New-Item -Times 1 -ParameterFilter { ($path -eq './tmp_tmp') -and ($ItemType -eq 'Directory') }
+        Assert-MockCalled Save-Module -Times 1 -Debug -ParameterFilter {
+          ($Name -eq 'SomeDSCModule') `
```

Comment:
If you move the `-and` to the end of this line you won't need to use backticks to indicate line continuation